health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/MapViewPopup.js

```
@@ -29,20 +29,26 @@ const ViewElement = ({ type, label }) => {
+            />
+            <path
+              d="M5 20V11.25L10.4 6L15.8 11.25V20H11.3V15.625H9.5V20H5ZM9.5 13.875H11.3V12.125H9.5V13.875ZM17.6 20V10.5281L12.9425 6H15.485L19.4 9.80625V20H17.6ZM21.2 20V9.08437L18.0275 6H20.57L23 8.3625V20H21.2Z"
               fill="white"
             />
           </svg>
         );
       case "Facility":
         return (
-          <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
-            <rect x="0.5" y="0.5" width="23" height="23" rx="3.5" fill="#C84C0E" stroke="#D6D5D4" />
+          <svg xmlns="http://www.w3.org/2000/svg" width="27" height="39" viewBox="0 0 27 39" fill="none">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)